src/wp-includes/blocks/template-part.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * Server-side rendering of the `core/template-part` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/template-part` block on the server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string The render.
+ */
```

Comment:
```suggestion
 * Renders the `core/template-part` block on the server.
 *
 * @since 5.9.0
 * 
 * @param array $attributes The block attributes.
 * @return string The render.
 */
```

Reply:
This will need to be changed in Gutenberg as this file is copied from `@wordpress/block-library`.

Reply:
> This will need to be changed in Gutenberg as this file is copied from `@wordpress/block-library`.

Indeed. AFAICS, we don't use `@since` there (or at least not with Core versions), which makes sense (since it'd be misleading within the context of the GB repo and project, and its own versions).